### PR TITLE
GEN 20 Project Loading Animation

### PR DESCRIPTION
**Pull request by vishalkantharaju**

### CODE REVIEW

**Review by Tanyk2004 — CHANGES_REQUESTED** [review]
Just two things:

- Can you add the loading animation to every page not just the what we do page? 
- Also when the images in the slideshow component are loading can we have a loading animation play there? ( probably use something along the lines of what Sahil showed us in shadcn )

**Comment by Tanyk2004** [comment]
Oh wait and one more thing, the button on the homepage is not colored anymore, is that by design or a bug? I think the old button with the gradient looked better imo. Thoughts?

image

**Review by Tanyk2004 — CHANGES_REQUESTED** [review]

File: src/styles/eclairbutton.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 
 .fancybutton {
-    --magic-rainbow-color-1: hsl(275, 100%, 73%);
+    --magic-rainbow-color-1: #c675ff
```
[the remark this answers is not in the document]
Comment (Tanyk2004) [reply src/styles/eclairbutton.css:4]: also probably make them both hexcodes or hsl values just for readibiliy.

[the remark this answers is not in the document]
Comment (vishalkantharaju) [reply src/styles/eclairbutton.css:4]: oh loll my bad

**Pull request merged**